SparkPost/2web2ui
# FE-878: Page Wrapper component

### What Changed
 - Added page wrapper component

### How To Test
 - Using appteam account, verify page component is using new component
 - Check that default version of app is not broken on pages

### To Do
